.github/workflows/main.yml

```diff
@@ -246,17 +246,10 @@ jobs:
         uses: actions/setup-node@b39b52d1213e96004bfcb1c61a8a6fa8ab84f3e8 # v4.0.1
         with:
           node-version: ${{ env.NODE_VERSION }}
-      - name: Set up Emscripten
-        uses: mymindstorm/setup-emsdk@6ab9eb1bda2574c4ddb79809fc9247783eaf9021 # v14
-        with:
-          version: ${{ env.EMSCRIPTEN_VERSION }}
-      - name: Build
+      - name: Install pyodide-build and Pyodide cross-build environment
         run: |
-          # TODO remove https://github.com/pyodide/pyodide/issues/5585
-          pip install -U wheel==0.45.1
-          pip install pyodide-build==$PYODIDE_VERSION
-          cd hypothesis-python/
-          CFLAGS=-g2 LDFLAGS=-g2 pyodide build
```

Comment:
The `CFLAGS=-g2 LDFLAGS=-g2 pyodide build` step is redundant, as:
1. We delegate `pyodide build` to `pypa/build` for building pure Python wheels, and the compilation flags won't do anything
2. We've been running `python -m build --wheel hypothesis-python --outdir dist/` anyway, which has been the wheel built by `pyodide build` in the previous

For pure Python packages, `pyodide build` does not provide any special provisions or invoke Pyodide-specific behaviour – using `pypa/build` is better.